css-canvas-medical/index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="nav">
+      <div class="column-header">
```

Comment:
You shouldn't have `column` classes named after what they are being applied to, they should be named after their size and only have `flex-basis` or `width` applied to them. Just like with the `row` classes I already mentioned, any additional styling should be applied to other classes you can add to the element in addition to the `column` class for setting its width.